src/NLog.StructuredLogging.Json/LoggerExtensions.cs

```diff
@@ -35,7 +35,7 @@ public static void ExtendedError(this ILogger logger, string message, object log
         }
 
         public static void Extended(this ILogger logger, LogLevel logLevel, string message, object logProperties,
-            Exception ex = null)
+            Exception ex = null, string tag = "")
```

Comment:
When this value is used (i.e. when there is an exception) it should always have a non-null value. The code will make a new guid if it's null.